tests/test_server.py

```diff
@@ -316,6 +317,23 @@ def version(self):
 
     print(MockServer(remote_server).version)
     with pytest.raises(
-        ValueError, match="Error connecting to DPF LegacyGrpcServer with version 1.0"
+            ValueError, match="Error connecting to DPF LegacyGrpcServer with version 1.0"
```

Comment:
A messages should always have a verb and concluding period. They should also be in plain English. Something like this would be better: "Could not connect to DPF legacy gRPC server with version 1.0."

Reply:
@PipKat , thanks @PipKat , we will fix this in the server side.